content/docs/introduction/comparison.md

```diff
@@ -244,43 +244,39 @@ blackbox probing is sufficient.
+The primary difference is that Sensu clients [register themselves](https://docs.sensu.io/sensu-core/latest/reference/clients),
+and can determine the checks to run either from central or local configuration. Sensu clients are also able to be automatically deregistered using a number of [plugins](https://github.com/sensu-plugins/sensu-plugins-sensu) and [integrations](https://docs.sensu.io/sensu-enterprise/latest/built-in-handlers/)
 
-There is also a [client socket](https://sensuapp.org/docs/0.27/reference/clients.html#what-is-the-sensu-client-socket) permitting arbitrary check results to be pushed into Sensu.
+There is also a [client socket](https://docs.sensu.io/sensu-core/latest/reference/clients/#what-is-the-sensu-client-socket) permitting ad-hoc check results to be pushed into Sensu. The client socket can also be used as a receiver for alerts generated via applications emitting JSON formatted data, [SNMP traps](https://docs.sensu.io/sensu-core/latest/guides/snmp-sensu-guide/), or serve as a way to [create proxy (just-in-time) clients](https://docs.sensu.io/sensu-core/1.4/reference/clients/#proxy-clients).
 
 ### Data model
 
-Sensu has the same rough data model as [Nagios](/docs/introduction/comparison/#prometheus-vs-nagios).
+Similar to Nagios, Sensu is host-based. The default [Sensu package](https://sensu.io/features/downloads) contains the [`sensu-api` process](https://docs.sensu.io/sensu-core/latest/api/overview/), which allows for programmatic interaction with Sensu. Sensu clients and check are also able make use of custom attributes, which function as tags and labels. These can be queried using the Sensu API.
+
+Some Sensu plugins (https://github.com/sensu-plugins) also perform contact routing-like functions.
 
 ### Storage
 
-Sensu has storage in Redis called stashes. These are used primarily for storing
-silences. It also stores all the clients that have registered with it.
+Sensu uses Redis to persist monitoring data, including the Sensu client registry, check results, check execution history, and current event data. By default, the last 21 check results are stored. Sensu provides a dashboard ([Uchiwa](https://uchiwa.io/#/)) for check data visualization.
 
 ### Architecture
 
-Sensu has a [number of components](https://sensuapp.org/docs/0.27/overview/architecture.html). It uses
+Sensu has a [number of components](https://docs.sensu.io/sensu-core/latest/overview/architecture/). It uses
 RabbitMQ as a transport, Redis for current state, and a separate server for
-processing.
+processing and API access. 
 
-Both RabbitMQ and Redis can be clustered. Multiple copies of the server can be
-run for scaling and redundancy.
+All components of a Sensu deployment (RabbitMQ, Redis, and Sensu Server/API) can be clustered for highly available and redundant configurations. The Sensu Server process, when in a clustered configuration, performs its own internal leader election for check request scheduling and event processing.
```

Comment:
No need to get into how exactly it works, that the feature exists is enough to mention.

Reply:
:heavy_check_mark: